scenarioo-server/build.gradle

```diff
@@ -81,6 +81,11 @@ dependencies {
 
     providedRuntime 'org.springframework.boot:spring-boot-starter-tomcat'
 
+
+    compile 'io.springfox:springfox-swagger2:2.9.2'
```

Comment:
do we need an explicit version?

Reply:
Of course - versions should always be explicit - we can discuss if you not agree

the question is actually: arent our other dependencies fixed versions? I think they should be!

I rather prefer manually updating versions in a well controlled manual fashion than taking the risk that somebody gets blocked because a new version brings in troubles --> non-deterministic builds (see also the "R" in CRISP build criterias!).

Reply:
the versions are fixed through bom files. If we specify a version explicitely it can break everything. That's the whole idea of the bom.

Reply:
remark: most of our other dependency versions are by the way as well fixed -see gradle.build file in the root of scenarioo that specifies these other versions - I can put this version here as well there - to have all in the same place.

Reply:
bom file?? we use gradle not maven, right?

Reply:
ahm I understand, ok, I see - yes would be a good idea, but we do not use BOM yet - is only supported since gradle 5 - we still use gradle 4. So for now we fix all our versions. Its already like that (just in the other file with the dependencies - I think it is better to have them well defined and update them manually from time to time - automatic update almost like always fails anyway!)

Reply:
I moved the version now to the dependency management part in the root gradle.build file, but it's still a fixed version, cause the current used dependency apüproach does not use proper BOM mechanism.